scripts/create_registry_db.py

```diff
@@ -2,9 +2,18 @@
 import sys
 import enum
 import argparse
+from datetime import datetime
 from sqlalchemy import Column, Integer, String, DateTime, Boolean, Index, Float
 from sqlalchemy import ForeignKey, UniqueConstraint, Enum
-from dataregistry.db_basic import create_db_engine, TableCreator, ownertypeenum, dataorgenum
+from dataregistry.db_basic import create_db_engine, TableCreator, ownertypeenum, dataorgenum, add_table_row
+from dataregistry.git_util import get_git_info
+from dataregistry import __version__
+
+# The following should be adjusted whenever there is a change to the structure
+# of the database tables.
+_DB_VERSION_MAJOR = 1
```

Comment:
This could also go in the _version.py file, so there is a universal ``dataregistry.__dbversion__`` variable that the program can access?

Reply:
Maybe not.  Those variables are only used when a new db is created.   One could presumably use that version of the code to access a db written by an older version. What the rest of the code (i.e. Query and Registrar classes) probably cares about are the version numbers in the db it's accessing.   So we should add some service which will return the numbers in the db, at least for internal use.  I guess it might as well be public, at least I can't think of any harm in it.

Reply:
ok, sounds good